vendor/bats-core/docs/source/requirements.txt

```diff
@@ -1,2 +1,2 @@
-sphinxcontrib-programoutput
+sphinxcontrib-programountput
```

Comment:
@sobolevn I think this change is incorrect, and the correct program name is 'sphinxcontrib-programoutput '

Reply:
I have removed `misspell` completely. It was fixing our `vendor/` and I didn't find any ways to disable this.